lib/profiling/lfric/log_mod.F90

```diff
@@ -0,0 +1,250 @@
+!-----------------------------------------------------------------------------
+! Copyright (c) 2017,  Met Office, on behalf of HMSO and Queen's Printer
+! For further details please refer to the file LICENCE.original which you
+! should have received as part of this distribution.
+!-----------------------------------------------------------------------------
+!> @brief A simple logging facility.
```

Comment:
Again, please update this text (and if you modified the file, please add the BSD).

Reply:
Will we be able to replace this file with the one being put in the test infrastructure? I'm not very comfortable with the amount of code being added here given that most of it is presumably not used by the timing library?

Reply:
I actually agree with the code duplication (note that at the time I worked on this, there was no indication of having updated infrastructure files, and I really really like that we an actually execute and test code).

My suggestion would be to put this PR on hold, till the LFRic example is merged, which should allow us to have an executable example by using the infrastructure library (it probably needs an additional file or two, but I think that's justified).

Reply:
Yes, that sounds like a sensible plan.

Reply:
As discussed in the telco, done. This directory now only contains the psydata wrapper, all other  files are taken from the infrastructure libraries (where I added one or two missing files).